kivy/uix/codeinput.py

```diff
@@ -52,7 +52,7 @@
 # TODO: color chooser for keywords/strings/...
 
 
-class CodeInput(CodeNavigationBehavior, TextInput):
+class CodeInput(CodeNavigationBehavior, EmacsBehavior, TextInput):
```

Comment:
@el-ethan, we would like to merge your work. Could you remove `EmacsBehavior, ` from this line?